src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -197,17 +197,37 @@
   end
 
   describe 'GET #subprojects' do
+    let(:apache2) { create(:project, name: 'Apache:Apache2') }
+    let(:apache2_subproject1) { create(:project, name: 'Apache:Apache2:TestSubproject') }
+    let(:apache2_subproject2) { create(:project, name: 'Apache:Apache2:TestSubproject2') }
+    let(:apache2_subprojects) {[apache2_subproject1, apache2_subproject2]}
```

Comment:
Please move this inside the ``context 'subprojects' do`` context as you only use it there!